pkg/lb/interface.go

```diff
@@ -0,0 +1,14 @@
+package lb
+
+type LB interface {
+	EnsureEntry() error
+	EnsureForwarder() error
+}
+
+type Entry interface {
+	Ensure() error
+}
+
+type Forwarder interface {
+	Ensure() error
+}
```

Comment:
How do these abstractions work? Why do they have the same method and What are the implemantations for them?

Reply:
It's just abstractions that an LB is combined by an entry and a forwarder. Coincidentally，they have the same function by now.

Reply:
Can you give examples for different/future implementation of these interfaces? I'm not sure if these are good abstractions for the LB.